cite.html

```diff
@@ -41,7 +41,7 @@ <h1>
 
             <p><b>1. Body text</b>
 	      <pre><code>
-Computations were done using the ASPECT code version 1.5.0, see \cite{heister_aspect_methods2,...}.</code></pre>
+Computations were done using the ASPECT code version 2.0.0, see \cite{heister_aspect_methods2,...}.</code></pre>
```

Comment:
We should really remove the `,...` here. That's just lazy.

Reply:
This is there on purpose. Can we discuss maybe during the hackathon?

Reply:
Yes. I think I know what you mean, though if you hope that a user puts something specific into these dots, then you should say what exactly right here. (Unless that comes next -- github just shows me this one line right now.)